### DISCUSSION TITLE
Use of weblate for translations

### DISCUSSION THREAD

**Discussion by JigSawFr** [astubenbord/paperless-mobile#37] [Feature Requests]
Maybe it can be interesting to use weblate to manage translation in a better way ? (Free for OpenSource)

**Comment by astubenbord** [comment]
Thanks, I'll take a look once the 250 keys are exceeded on Localizely. Until that happens, it also stays free.

**Reply by JigSawFr** [reply]
NP, I suggest it mostly for automation, integration with GH and open to public contributions

**Comment by TurtleIdiot** [comment]
I heard crowdin is pretty good for localisation

**Reply by astubenbord** [reply]
Hi, I just switched the localizations to Crowdin (see README)
